index.js

```diff
@@ -77,7 +77,10 @@ async function register (server, options) {
       request.logger = nullLogger
       return h.continue
     }
-    request.logger = logger.child({ req: request })
+
+    const childBindings = options.getChildBindings ? options.getChildBindings(request) : { req: request }
```

Comment:
Would you mind moving `getChildBindings` to a variable at an upper level?

Reply:
done